English/README.md

```diff
@@ -84,36 +143,61 @@
-    * Persistent python objects in database with SQLAlchemy (PostgreSQL, AWS Aurora).
-    * Use unittests with Docker and Mocks.
-  * **Time Series Databases**: [InfluxDB](https://www.influxdata.com/).
-  * **Document Databases**: [MongoDB Atlas](https://www.mongodb.com/cloud/atlas/lp/try2?utm_source=google&utm_campaign=gs_americas_colombia_search_brand_atlas_desktop&utm_term=mongodb%20atlas&utm_medium=cpc_paid_search&utm_ad=e&gclid=CjwKCAjw4KD0BRBUEiwA7MFNTaXgR5gxbpm9vmvLtPLnBkdaqBr9fWxMFSpO9orpBlAC0zC0R5RAGBoCnVEQAvD_BwE).
-  * **Key-value store**: [AWS Dynamodb](https://aws.amazon.com/es/dynamodb/).
-  * **BONUS - Graph Databases**: [Neo4j](https://neo4j.com/) && [OrientDB](https://orientdb.com/).
-
-* **Logs and Data Visualization**
-  * Setting up InfluxDB and Grafana.
-  * Creating dashboards.
-  * Realtime stats generation from an analytics service.
-
-* **CI/CD**
-  * Defining development workflows (test, delivery, deployment, rollbacks, patchs, vulnerability scan).
-  * Implement code scripts for pipeline stages.
-  * Use Github Actions for CI/CD implementation.
-  * **ChatOps**: Notificaciones y alertas.
-
-* **Monitoring**
-  * Stack EFK.
-  * AWS CloudWatch.
-
-* **Profiling**
-  * Basic security configuration steps before release delivery.
+
+* Polyglot Persistence layer && Not-Only SQL (NoSQL):
+  * **Relational Database modeling**
+    * Principles and best practices.
+  * **Time Series Databases**:
+    * [InfluxDB](https://www.influxdata.com/).
+  * **Document Databases**: 
+    * [MongoDB Atlas](https://www.mongodb.com/cloud/atlas/lp/try2?utm_source=google&utm_campaign=gs_americas_colombia_search_brand_atlas_desktop&utm_term=mongodb%20atlas&utm_medium=cpc_paid_search&utm_ad=e&gclid=CjwKCAjw4KD0BRBUEiwA7MFNTaXgR5gxbpm9vmvLtPLnBkdaqBr9fWxMFSpO9orpBlAC0zC0R5RAGBoCnVEQAvD_BwE).
+  * **Key-value store**: 
+    * [AWS Dynamodb](https://aws.amazon.com/es/dynamodb/).
+  * **Graph-oriented databases**:
+    * [Neo4j](https://neo4j.com/) && [OrientDB](https://orientdb.com/).
+  * Unit testing using virtual containers.
+
+* Visualization
+  * Setting the environment using [InfluxDB](https://www.influxdata.com/) and [Grafana](https://grafana.com/).
+  * Creation of information boards.
+  * Generation of real-time statistics.
+  * Considerations for a production deployment.
+
+* Continuous Integration and Continuous Deployment (CI/CD)
+  * Defining development workflows
+    * Testing stack.
+    * Continuous Delivery.
+    * Continuous Deployment.
+    * Rollbacks.
+    * Patching.
+    * Vulnerability scanning.
+    * Request For Changes (RFC) validation before deployment.
+  * Continuous Integration tools
+    * [Travis CI](https://travis-ci.org/).
+    * GitHub Actions.
+    * [DroneIO](https://cloud.drone.io/).
+    * [CircleCI](https://circleci.com/).
+    * [AWS CodePipeline](https://aws.amazon.com/es/codepipeline/).
+  * Implement Python code scripts for pipeline stages.
+  * **ChatOps implementation**
+    * [Microsoft Teams]().
```

Comment:
Could you please add discord here, I will add it to the Spanish version later

Reply:
Done.